src/app/loginComponents/accounts/controls/delete-account-dialog/delete-account-dialog.component.spec.ts

```diff
@@ -16,9 +16,9 @@ describe('DeleteAccountDialogComponent', () => {
   beforeEach(async(() => {
     TestBed.configureTestingModule({
       declarations: [DeleteAccountDialogComponent],
+      schemas: [NO_ERRORS_SCHEMA],
```

Comment:
Is this a good idea :thinking:

Reply:
It's to ignore sub components in testing because we're not interested in those.

Reply:
Could you add this to the style guide?